pkg/storage/tpr/storage_interface.go

```diff
@@ -106,8 +107,26 @@ func (t *store) Create(
 		t.singularKind.URLName(),
 	).Body(data)
 
+	res := req.Do()
+	if res.Error() != nil {
+		glog.Errorf("executing POST for %s/%s (%s)", ns, name, res.Error())
+	}
+	var statusCode int
+	res.StatusCode(&statusCode)
+	if statusCode == http.StatusConflict {
+		return storage.NewKeyExistsError(key, 0)
+	}
+	if statusCode != http.StatusCreated {
+		return fmt.Errorf(
+			"executing POST for %s/%s, received response code %d",
```

Comment:
It's not clear to me from the message that this represents an error - I think I would probably have missed this looking at the log.  Will you add `Error` to the text of the message?